### PR TITLE
wicket-jquery-ui is made compatible with latest wicket-8

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	</scm>
 
 	<properties>
-		<wicket.version>8.0.0-M3</wicket.version>
+		<wicket.version>8.0.0-SNAPSHOT</wicket.version>
```
Comment (martin-g) [inline pom.xml]: Is there a `<repository>` definition in this pom.xml for -SNAPSHOTs ?
If there is no such then the build at CI will fail!
Not everyone uses cutting edge like you, @solomax ! ;-)

Comment (solomax) [inline pom.xml]: This is why I need M4 :)))

This is SNAPSHOT build, I believe it might have SNAPSHOT dependencies ...
The Travis CI build passed ....

Comment (martin-g) [inline pom.xml]: If Travis is happy then I'm happy too! :-)

Comment (solomax) [inline pom.xml]: Thanks!
Can't wait M4 :))
Our release is blocked by this WebSocket/Session issue :(

**Comment by sebfz1** [comment]
Hi Maxim & Martin! :)

@solomax , thanks for your PR but it was already available (made by Andrea) :)
I will merge your as it is the most recent, I just have to port a change afterward:

- [ ] replace `new JSONObject(object).toString()` to `JsonUtils.toJson(object)`

```java
public static String toJson(Object object) {
	try {
		return new JSONObject(object).toString();
	} catch (InvocationTargetException | IllegalAccessException | IntrospectionException e) {
		throw new RuntimeException(e);
	}
}
```

The snapshot should be available in a few minutes (automatically deployed by Travis)

**Comment by solomax** [comment]
Sorry :((
Haven't notices there is PR already :((

**Comment by sebfz1** [comment]
That's fine don't worry :) You just made something that used to be already made ;)

**Comment by solomax** [comment]
@sebfz1 Maybe it would be possible to create fresh SNAPSHOT? So we can test both wicket and wicket-jquery-ui? :)

**Comment by sebfz1** [comment]
Damned! It is supposed to be automatically deployed! But I have some javadocs warnings which are failing the snapshot/deploy build! A little bit rude... I'm correcting this ASAP

**Comment by sebfz1** [comment]
The snapshot should appear here in a few minutes:
https://oss.sonatype.org/content/repositories/snapshots/com/googlecode/wicket-jquery-ui/wicket-jquery-ui-core/8.0.0-SNAPSHOT/

**Comment by sebfz1** [comment]
Deployed!

**Comment by solomax** [comment]
Thanks!

**Pull request merged**